src/pudl/metadata/classes.py

```diff
@@ -135,12 +135,15 @@ def _format_for_sql(x: Any, identifier: bool = False) -> str:  # noqa: C901
     return f"'{x}'"
 
 
-JINJA_ENVIRONMENT: jinja2.Environment = jinja2.Environment(
-    loader=jinja2.FileSystemLoader(
-        os.path.join(os.path.dirname(__file__), "templates")
-    ),
-    autoescape=True,
-)
+def _get_jinja_environment(template_dir: DirectoryPath = None):
+    if template_dir:
+        path = template_dir / "templates"
+    else:
+        path = Path(__file__) / "templates"
```

Comment:
I still have the `metadata.yml.jinja` template for Datasette living in the templates in this directory because it didn't really feel like it belonged in docs. But maybe I should just move it over as well.

Reply:
We'll definitely have other metadata exports to create, and I don't think they should live in docs, so I think this is a fine place for it to stay.  I think it would be helpful to clarify which metadata it is though. Could the template be called `datasette-metadata.yml.jinja` instead (even though the file that gets created will be `metadata.yml`)?

At the very least we'll also need to export metadata for the PUDL data catalog.

Reply:
Good idea. I updated the template to `datasette-metadata.yml.jinja` as well as changed the script name to `datasette_metadata_to_yml`